openshift/templates/dbbackup.dc.json

```diff
@@ -74,6 +108,17 @@
                 "name": "${NAME}:${VERSION}"
               }
             }
+          },
+          {
+            "type": "ImageChange",
+            "imageChangeParams": {
+              "automatic": true,
+              "containerNames": ["flyway-migration"],
+              "from": {
+                "kind": "ImageStreamTag",
+                "name": "${FLYWAY_NAME}:${VERSION}"
+              }
+            }
```

Comment:
This will mean any SQL change will cause dbbackup to redeploy. 

@pdittaro What if we merged in a PR while this pod was in the middle of backing up and shipping date to the store? Something noteworthy. can we disable PR's from midnight-1am?